src/core/loader/ncch.cpp

```diff
@@ -286,6 +286,7 @@ bool NCCH::parseSMDH(const std::vector<u8>& smdh) {
 	} else if (taiwan) {
 		region = Regions::Taiwan;
 	}
+	return true;
```

Comment:
The weirdest thing. I sometimes hit an illegal instruction around here for a bit. If that is the UB that this was addressing.

Reply:
If it's a recent thing then probaby. This was added in #179